english/forest/drag/roaming_spirit/beer_sandwich/new_friend/new_friend.md

```diff
@@ -1 +1,4 @@
-You tell him that you know about an old tale that says that spirits can interact with living creatures
+You tell him that you know about an old tale that says that spirits can interact with living creatures. Having said that, you try to
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.